src/main/rx/spektrum.c

```diff
@@ -293,6 +435,21 @@ static uint8_t spektrumFrameStatus(void)
         }
     }
 
+ #ifdef USE_SPEKTRUM_VTX_CONTROL
```

Comment:
Could be `#if defined(USE_SPEKTRUM_VTX_CONTROL) && defined(VTX_COMMON)`.

Reply:
Many ways to skin a cat. Is that more readable,?

Reply:
I think so. Also, it's consistent with how it's done in the majority of cases in the rest of the codebase.